MyStore/src/include/JsonObject.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <vector>
+#include <unordered_map>
+#include <fstream>
+#include <string>
+#include <any>
+
+// JsonObject class
+// usage example:
+// 
+// JsonObject json;
+// bool b = json.LoadFromFile("..\\MyStore\\data\\Data.json");
+// 
+// int count = json["products"].GetArraySize();
+// for (int i = 0; i < count; ++i)
+// {
+// 	JsonObject child = json["products"][i];
+// 	std::cout << child["name"].AsString() << ", " << child["color"].AsString() << ", " << child["price"].AsInt() << std::endl;
+// }
```

Comment:
Yes. For clarity and convenience I added project folder variable to MyStore.cpp with example of usage. Try to use it with appended path instead of fiddling with relative paths. Relative path can be resolved to different absolute paths when passed to different modules.